### ISSUE TITLE
v3+: Can we use "node:<buildinName>" as import specifier in v3+?

### ISSUE THREAD

**Issue by frank-dspeed** [rollup/rollup#4595]
### Expected Behavior / Situation

i type something like 

```
cat my.js | grep "node:"
```

and get all nodejs buildin modules that are required by the module

### Actual Behavior / Situation

i write the command and i get nothing 

### Modification Proposal

```js
import { readFile } from "fs/promises";
// should be for example
import { readFile } from "node:fs/promises";
```

## Target Branch
- https://github.com/rollup/rollup/pull/4549

**Comment by lukastaegert** [comment]
I put it into #4596

**Comment by lukastaegert** [comment]
This issue has been resolved via #4596 as part of rollup@3.0.0-4. Note that this is a pre-release, so to test it, you need to install Rollup via `npm install rollup@3.0.0-4` or `npm install rollup@beta`. It will likely become part of a regular release later.
